src/Paket/Commands.fs

```diff
@@ -372,6 +384,7 @@ type Command =
     | [<CustomCommandLine("find-packages")>]            FindPackages of ParseResults<FindPackagesArgs>
     | [<CustomCommandLine("find-package-versions")>]    FindPackageVersions of ParseResults<FindPackageVersionsArgs>
     | [<CustomCommandLine("fix-nuspec")>]               FixNuspec of ParseResults<FixNuspecArgs>
+    | [<CustomCommandLine("generate-nuspec")>]          GenerateNuspec of ParseResults<GenerateNuspecArgs>
```

Comment:
- [Nuspec Schema](https://docs.microsoft.com/en-us/nuget/schema/nuspec)
The generated nuspec is needed to enable `pack` to work. We're currently intercepting the nuspec that nuget generates and then trimming its dependencies to make a usable package.